### ISSUE TITLE
5章の00_sin_simple_rnn_tensorflow.py プログラムに関して

### ISSUE THREAD

**Issue by embeddedsamurai** [yusugomori/deeplearning-tensorflow-keras#1]
このプログラムの

``# 最後の時系列データから未来を予測``
z_ = Z[-1:]
y_ = y.eval(session=sess, feed_dict={
            x: Z[-1:],
            n_batch: 1
})
部位ですが, 本来意図していたことは

z_ = Z[:,::-1,:]
y_ = y.eval(session=sess, feed_dict={
            x: Z[:,::-1,:],
            n_batch: 1
})
にして最後のデータに対して評価をしたかったのだと
思うのですが，認識あっていますでしょうか。

**Comment by embeddedsamurai** [comment]
すみません。私の認識が間違っていました。

Zは176個の束の　一つ目の束の25個のデータであり,
このデータからy_を計算したあと, y_を最新データとして
最初の1つのデータを捨てて最後にy_を加え25個のデータを更新して, 
それを176回繰り返すプログラムだということを理解しました。

Zは最後で
Z = np.append(Z, sequence_, axis=0)
で更新されたあと, ループの最初に戻り
z_ = Z[-1:]
されて　y_で更新された 25個のデータを取得するために行われていることを理解しました。
ですので
sequence_ = np.concatenate((z_.reshape(maxlen, n_in)[1:], y_), axis=0).reshape(1, maxlen, n_in)
Z = sequence_
として計算しても構わないことも理解しました。

問題は解決しました。
